src/modules/db_redis/redis_connection.c

```diff
@@ -137,14 +140,23 @@ int db_redis_connect(km_redis_con_t *con) {
     char hosts[MAX_URL_LENGTH];
     char* host_begin;
     char* host_end;
+    redisSSLContext *ssl = NULL;
```

Comment:
consider moving this parameter definition to the beginning of the function, as its needed in #if and #else